resources/definitions/ultimaker_sketch.def.json

```diff
@@ -1,6 +1,6 @@
 {
     "version": 2,
-    "name": "UltiMaker Sketch",
+    "name": "MakerBot Sketch",
```

Comment:
Please consider renaming the file, this is quite confusing (same for the others)

Reply:
Hi Erwan,

Yes, it should be MakerBot Sketch. 
It was changed recently, because the PM & Marketing for education wanted it this way..

Reply:
Ok, then the file should be called `makerbot_sketch.def.json`, or do you prefer to keep it this way ?

Reply:
I prefer it this way. I like to keep the Ultimaker machines as a group. These files are not user facing so it is OK to keep the name ultimaker_sketch.def.json